src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -1560,10 +1560,12 @@ private void DisConnectFromSignals()
         [EditorBrowsable(EditorBrowsableState.Never)]
         protected virtual void InitializeStyle(ViewStyle style = null)
         {
-#if PROFILE_TV
-            // tv profile doesn't use NUI Style, so do nothing and just return here!
-            return;
-#endif
+            if (style == null && ThemeManager.InitialThemeDisabled)
```

Comment:
Is ok if style is not null and ThemeManager.InitialThemeDisabled is false?

Reply:
For example, 
```
var Button = new Button(new ButtonStyle() {
/* style description */
});
```
In this case, the `style` is not null.
If it is a TV application, `InitialThemeDisabled` is false.

The code should not pass the style processing in this condition for cases like this.

Reply:
oh right! This patch is to apply style for this case!